aws/cloudformation/metaflow-cfn-template.yml

```diff
@@ -315,6 +372,88 @@ Resources:
         - ContainerName: !FindInMap ['ServiceInfo', 'ServiceName', 'value']
           ContainerPort: !FindInMap ['ServiceInfo', 'ContainerPort', 'value']
           TargetGroupArn: !Ref 'NLBTargetGroup'
+        - ContainerName: !FindInMap ['ServiceInfo', 'ServiceName', 'value']
+          ContainerPort: 8082
+          TargetGroupArn: !Ref 'NLBTargetGroupDBMigrate'
+  LambdaECSExecuteRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          Effect: Allow
+          Principal:
+            Service:
+              - lambda.amazonaws.com
+          Action:
+            - sts:AssumeRole
+      Path: /
+      Policies:
+        - PolicyName: Cloudwatch
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+            - Sid: CreateLogGroup
+              Effect: Allow
+              Action: logs:CreateLogGroup
+              Resource: !Join [ "", [ 'arn:aws:logs:', !Ref 'AWS::Region', ':', !Ref 'AWS::AccountId', ':*' ] ]
+            - Sid: LogEvents
+              Effect: Allow
+              Action:
+                - logs:PutLogEvents
+                - logs:CreateLogStream
+              Resource: !Join [ "", [ 'arn:aws:logs:', !Ref 'AWS::Region', ':', !Ref 'AWS::AccountId', ':log-group:/aws/lambda/', !Ref 'AWS::StackName', '-migrate-db:*'] ]
+        - PolicyName: VPC
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+            - Sid: NetInts
+              Effect: Allow
+              Action:
+                - ec2:CreateNetworkInterface
+                - ec2:DescribeNetworkInterfaces
+                - ec2:DeleteNetworkInterface
+              Resource: '*'
+  ExecuteDBMigration:
+    Type: AWS::Lambda::Function
+    Properties:
+      Runtime: python3.7
+      Role: !GetAtt LambdaECSExecuteRole.Arn
+      Handler: index.handler
+      Environment:
+        Variables:
+          MD_LB_ADDRESS: !Join ['', ['http://', !GetAtt 'NLB.DNSName', ':8082'] ]
+      Code:
+        ZipFile: |
+          import os, json
+          from urllib import request
+          
+          def handler(event, context):
+            response = {}
+            status_endpoint = "{}/db_schema_status".format(os.environ.get('MD_LB_ADDRESS'))
+            upgrade_endpoint = "{}/upgrade".format(os.environ.get('MD_LB_ADDRESS'))
+            
+            with request.urlopen(status_endpoint) as status:
+              response['init-status'] = json.loads(status.read())
+            
+            upgrade_patch = request.Request(upgrade_endpoint, method='PATCH')
+            with request.urlopen(upgrade_patch) as upgrade:
+              response['upgrade-result'] = upgrade.read().decode()
+              
+            with request.urlopen(status_endpoint) as status:
+              response['final-status'] = json.loads(status.read())
+          
+            print(response)
+            return(response)
```

Comment:
@savingoyal @ferras This Lambda Function is probably deserving of a quick review to make sure its functionality is acceptable for a user-facing workflow.  Basically it's just a return of "initial status", "upgrade result", and "final status" printed to Cloudwatch for posterity.